front-end-admin/src/components/callout/CalloutInfo.vue

```diff
@@ -1,54 +1,86 @@
 <template>
   <div class="jumbotron">
     <div class="container">
-      <h2>{{callout.title}}</h2>
-      <!-- TODO: List out associated needs with a specific callout -->
-      <h2>Update Callout Information</h2>
-      <form v-on:submit.prevent="onFormSubmit">
-        <div class="form-group">
-          <label class="control-label col-sm-2">Title:</label>
-          <input v-model="callout.name" type="text" width="50" required>
+      <br>
+      <br>
+      <b-card title="Edit Callout">
+      <b-card :title="callout.name">
+        <b-form @submit="onFormSubmit" class="form-horizontal" >
+          <input type="hidden" name="callout_id" v-model="callout.id">
+          <h5>Callout Name: </h5>
+          <b-form-textarea v-model="callout.name"
+                           required
+                           placeholder="Enter name.."
+                           :rows="1"
+                           no-resize
+                           name="name"
+                           :max-rows="1">
+          </b-form-textarea>
+          <h5>Description: </h5>
+          <b-form-textarea v-model="callout.descriptionMessage"
+                           required
+                           placeholder="Enter description.."
+                           :rows="3"
+                           name="Amazon URL"
+                           :max-rows="3">
+          </b-form-textarea>
+          <h5>Set End Date: </h5>
+          <br>
+          <b-button type="submit" variant="primary">Submit</b-button>
```

Comment:
Instead of Primary button use a Success button as that indicates a submission in Bootstrap

Reply:
I changed this, on the documentation they were using primary, https://bootstrap-vue.js.org/docs/components/form, but I agree green might be more intuitive